modules/apps/batch-planner/batch-planner-test/src/testFunctional/tests/ImportObjectDefinitions.testcase

```diff
@@ -0,0 +1,200 @@
+				entityType = "ObjectDefinition (v1_0 - Liferay Object Admin REST)",
+				fileName = "student_and_teacher_objectDefinition_import.json");
+		}
+
+		task ("Then the new object definition is created") {
+			ObjectAdmin.openObjectAdmin();
+
+			AssertElementPresent(
+				key_labelName = "Teacher",
+				locator1 = "CreateObject#OBJECT_NAME");
+		}
+
+		task ("And student object definition is updated") {
+			ObjectAdmin.openObjectAdmin();
+
+			ObjectPortlet.selectCustomObject(label = "Student");
+
+			ObjectAdmin.goToFieldsTab();
+
+			ObjectAdmin.assertObjectField(
+				key_fieldLabel = "additional",
+				key_fieldType = "Text");
+		}
+	}
+
+	@priority = 4
+	test CanImportJSONFileWithOnlyNewRecordsImportStrategy {
+		property portal.acceptance = "true";
+
+		task ("Given JSON file with an existing object definition valid data and a new object definition valid data") {
+			ObjectDefinitionAPI.createAndPublishObjectDefinition(
+				en_US_label = "Student",
+				en_US_plural_label = "Student",
+				name = "Student",
+				requiredStringFieldName = "name");
+		}
+
+		task ("When I import with select Only Add New Records in Import Strategy and uncheck Stop the Import on Error") {
+			ImportExport.importFile(
+				entityType = "ObjectDefinition (v1_0 - Liferay Object Admin REST)",
+				fileName = "student_and_teacher_objectDefinition_import.json",
+				stopImportOnError = "false");
+		}
+
+		task ("Then the new object definition is imported successfully") {
+			ObjectAdmin.openObjectAdmin();
+
+			AssertElementPresent(
+				key_labelName = "Teacher",
+				locator1 = "CreateObject#OBJECT_NAME");
+		}
+	}
+
+	@priority = 4
+	test CanImportJSONLFile {
+		property portal.acceptance = "true";
+
+		task ("When I use import/export center to import the object definition with JSONL file") {
+			ImportExport.importFile(
+				entityType = "ObjectDefinition (v1_0 - Liferay Object Admin REST)",
+				fileName = "student_objectDefinition_import.jsonl");
+		}
+
+		task ("Then the object definition can be imported to the portal successful") {
+			ObjectAdmin.openObjectAdmin();
+
+			AssertElementPresent(
+				key_labelName = "Student",
+				locator1 = "CreateObject#OBJECT_NAME");
+		}
+	}
+
+	@priority = 4
+	test CanImportObjectDefinitionWithExportedJSONLFile {
+		property portal.acceptance = "true";
+
+		task ("Given a student object with a custom field name") {
+			ObjectDefinitionAPI.createAndPublishObjectDefinition(
+				en_US_label = "Student",
+				en_US_plural_label = "Student",
+				name = "Student",
+				requiredStringFieldName = "name");
+		}
+
+		task ("And I export ObjectDefinition to a JSONL format file in Import/Export center") {
+			ImportExport.exportAndUnzipFile(
+				entityType = "ObjectDefinition (v1_0 - Liferay Object Admin REST)",
+				exportFields = "externalReferenceCode,label,name,objectFields,pluralLabel,scope,status",
+				exportFileFormat = "JSONL");
+		}
+
+		task ("And delete the student object from the portal") {
```

Comment:
The codes are correct, but according to a pr Kevin did recently, Meg shared that we can skip these steps, that are only for manual testing, we can upload a dependency file in automation test, so we can
- remove the upper Given steps
- upload file from dependencies folder then remove the unused temFile parameter

Reply:
for this test name 'CanImportObjectDefinitionWithExportedJSONLFile', if we only want to test import there, can we remove this test? because if we remove the exported file there, the test is the same with the 'CanImportJSONLFile'
cc @yanan

Reply:
good catch, that is right, sorry I just noticed it. Then we can leave it, thanks!